fossunited/fossunited/notification/rsvp_confirmation_email/rsvp_confirmation_email.md

```diff
@@ -0,0 +1 @@
+<h1>Thank you for RSVPing</h1>
```

Comment:
HTML in .md file? Maybe you wanted to have a Jinja template (inside templates folder of the app)